README.md

```diff
@@ -34,15 +34,12 @@ Add the following to the ```Webservice``` section of your application config.
 ### Controller
 
 ```php
-<?php
-
 namespace App\Controller;
 
 use Cake\Event\Event;
 
 class StatusesController extends AppController
 {
-
```

Comment:
Classes should have a empty line at the start: https://github.com/cakephp/cakephp/blob/master/src/Controller/ErrorController.php#L27